File: Python/Program/Aksesoris.py
from PetShop import PetShop

class Aksesoris(PetShop):
    def __init__(self, id, nama, harga, stok, jenis, bahan, warna):
        super().__init__(id, nama, harga, stok)
        self.__jenis = jenis
        self.__bahan = bahan
        self.__warna = warna

    def setJenis(self, jenis):
        self.__jenis = jenis
    def getJenis(self):
        return self.__jenis
    
    def setBahan(self, bahan):
        self.__bahan = bahan
    def getBahan(self):
        return self.__bahan
    
    def setWarna(self, warna):
        self.__warna = warna
    def getWarna(self):
        return self.__warna
    

File: Java/Program/Aksesoris.java
class Aksesoris extends PetShop{
    private String jenis;
    private String bahan;
    private String warna;

    Aksesoris(String id, String nama, int harga, int stok){
        super(id, nama, harga, stok);
        this.jenis = "";
        this.bahan = "";
        this.warna = "";
    }

    public void setJenis(String jenis){
        this.jenis = jenis;
    }
    public String getJenis(){
        return this.jenis;
    }

    public void setBahan(String bahan){
        this.bahan = bahan;
    }
    public String getBahan(){
        return this.bahan;
    }

    public void setWarna(String warna){
        this.warna = warna;
    }
    public String getWarna(){
        return this.warna;
    }
}